scripts/update-arb-from-crowdin.mjs

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/home/julien/Downloads/node-v20.14.0-linux-x64/bin/node
```

Comment:
Oops! the `/usr/bin/env node` shebang doesn't work for you? Still it's the most common. But yours will certainly not work on other machines ;)

Reply:
Good catch ! I need to be more careful with my commits, I am going to commit sensitive information one of these days.

I installed nodejs on my machine to run this script but I don't want to keep it so I didn't bother adding it to the path variable.